test/integration/orm_action_execution.go

```diff
@@ -0,0 +1,216 @@
+
+		})
+
+	})
+
+	Describe("ORM test for cluster scope", func() {
+		It("should match with the new CR", func() {
+			// 1. Get the deployment
+			dep, err := kubeClient.AppsV1().Deployments(ormtest_namespace).Get(context.TODO(), clusterscope_operand_deployment_name, metav1.GetOptions{})
+			framework.ExpectNoError(err, "Error gettting the deployment for ORM test")
+
+			// 2. Get the CR and backup the resource value
+			operatorCR, err := dynamicClient.Resource(groupVersionRes4clusterscope).Namespace(v1.NamespaceAll).Get(context.TODO(), clusterscope_operator_cr_name, metav1.GetOptions{})
+			framework.ExpectNoError(err, "Error gettting the operator CR")
+			origCRValue4ClusterScope, _, err = util.NestedField(operatorCR, "destPath", clusterscope_operand_path_in_cr)
+
+			// 3. Build and execute resize action on the deployment
+			targetSE := newResizeWorkloadControllerTargetSE(dep)
+			resizeupAction, desiredPodSpec := newResizeActionExecutionDTO(targetSE, ORM_RESIZE_DOWN, &dep.Spec.Template.Spec)
+			_, err = actionHandler.ExecuteAction(resizeupAction, nil, &mockProgressTrack{})
+			if err != nil {
+				framework.Failf("Failed to execute resizing action for ORM cluster case")
+			}
+
+			// 4. Set the revert flag for afterEach
+			needRevertCR4ClusterScope = true
+
+			// 5. Wait and check the result
+			containerIdx, err := findContainerIdxInPodSpecByName(desiredPodSpec, clusterscope_operand_container_name)
+			framework.ExpectNoError(err, "Can't find the container in the pod's spec")
+			waitForOperatorCRToUpdateResource(dynamicClient, groupVersionRes4clusterscope, v1.NamespaceAll, clusterscope_operator_cr_name, clusterscope_operand_path_in_cr, &desiredPodSpec.Containers[containerIdx].Resources)
+		})
+	})
+
+	AfterEach(func() {
+		if needRevertCR4NsScope {
+			err := revertOperatorCR(dynamicClient, groupVersionRes4nsscope, ormtest_namespace, namespacescope_operator_cr_name, namespacescope_operand_path_in_cr, origCRValue4NsScope)
+			framework.ExpectNoError(err, "Error reverting the namespace scope operator CR")
+		}
+		if needRevertCR4ClusterScope {
+			err := revertOperatorCR(dynamicClient, groupVersionRes4clusterscope, v1.NamespaceAll, clusterscope_operator_cr_name, clusterscope_operand_path_in_cr, origCRValue4ClusterScope)
+			framework.ExpectNoError(err, "Error reverting the cluster scope operator CR")
+		}
+	})
+})
+
+func revertOperatorCR(client dynamic.Interface, groupVersionRes schema.GroupVersionResource, nsName, crName, destPath string, value interface{}) error {
+	operatorCR, err := client.Resource(groupVersionRes).Namespace(nsName).Get(context.TODO(), crName, metav1.GetOptions{})
+	framework.ExpectNoError(err, "Error gettting the operator CR")
+	fields := strings.Split(destPath, ".")
+	err = util.SetNestedField(operatorCR.Object, value, fields[1:]...)
+	dynResourceClient := client.Resource(groupVersionRes).Namespace(nsName)
+	_, err = dynResourceClient.Update(context.TODO(), operatorCR, metav1.UpdateOptions{})
```

Comment:
Why break line 186 and 187 into two different lines?

Reply:
will combine them together

Reply:
done